tests/v2/validation/provisioning/provisioning_node_driver_test.go

```diff
@@ -403,163 +242,25 @@ func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning_RKE2AWSClusterDyn
-
-					checkFunc := clusters.IsProvisioningClusterReady
-
-					err = wait.WatchWait(result, checkFunc)
-					assert.NoError(r.T(), err)
-					assert.Equal(r.T(), clusterName, clusterResp.Name)
-
-				})
-			}
-		}
+func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning() {
+	for provider_name := range r.providers {
+		fmt.Printf("Starting test for %v\n", r.providers[provider_name])
+		provider_struct := CreateProvider(r.providers[provider_name])
+		r.Provisioning_RKE2Cluster(provider_struct)
 	}
 }
 
-func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning_RKE2HarvesterClusterDynamicInput() {
+func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioningDynamicInput() {
 	nodesAndRoles := NodesAndRolesInput()
+	fmt.Printf("%v", nodesAndRoles)
 	if len(nodesAndRoles) == 0 {
 		r.T().Skip()
 	}
 
-	subSession := r.session.NewSession()
-	defer subSession.Cleanup()
-
-	client, err := r.client.WithSession(subSession)
-	require.NoError(r.T(), err)
-
-	cloudCredential, err := harvester.CreateHarvesterCloudCredentials(client)
-	require.NoError(r.T(), err)
-
-	tests := []struct {
-		name   string
-		client *rancher.Client
-	}{
-		{"Admin User", r.client},
-		{"Standard User", r.standardUserClient},
-	}
-
-	var name string
-	for _, tt := range tests {
-		for _, kubeVersion := range r.kubernetesVersions {
-			name = tt.name + " Kubernetes version: " + kubeVersion
-			for _, cni := range r.cnis {
-				name += " cni: " + cni
-				r.Run(name, func() {
-					testSession := session.NewSession(r.T())
-					defer testSession.Cleanup()
-
-					testSessionClient, err := tt.client.WithSession(testSession)
-					require.NoError(r.T(), err)
-
-					clusterName := AppendRandomString(baseHarvesterClusterName)
-					generatedPoolName := fmt.Sprintf("nc-%s-pool1-", clusterName)
-					machinePoolConfig := machinepools.NewHarvesterMachineConfig(generatedPoolName, namespace)
-
-					machineConfigResp, err := machinepools.CreateMachineConfig(machinepools.HarvesterResourceConfig, machinePoolConfig, testSessionClient)
-					require.NoError(r.T(), err)
-
-					machinePools := machinepools.RKEMachinePoolSetup(nodesAndRoles, machineConfigResp)
-
-					cluster := clusters.NewRKE2ClusterConfig(clusterName, namespace, cni, cloudCredential.ID, kubeVersion, machinePools)
-
-					clusterResp, err := clusters.CreateRKE2Cluster(testSessionClient, cluster)
-					require.NoError(r.T(), err)
-
-					result, err := testSessionClient.Provisioning.Clusters(namespace).Watch(context.TODO(), metav1.ListOptions{
-						FieldSelector:  "metadata.name=" + clusterName,
-						TimeoutSeconds: &defaults.WatchTimeoutSeconds,
-					})
-					require.NoError(r.T(), err)
-
-					checkFunc := clusters.IsProvisioningClusterReady
-
-					err = wait.WatchWait(result, checkFunc)
-					assert.NoError(r.T(), err)
-					assert.Equal(r.T(), clusterName, clusterResp.Name)
-
-				})
-			}
-		}
+	for provider_name := range r.providers {
+		fmt.Printf("Starting test for %v\n", r.providers[provider_name])
```

Comment:
Same as above make this a log statement instead. Actually maybe have all these print/log statements, might be a little much. What does the output look like for a standard test run?